components/Link/src/link.css

```diff
@@ -24,25 +24,25 @@
 
 .denhaag-link--focus,
 .denhaag-link:focus {
-  color: var(--denhaag-link-focus-color);
+  --denhaag-link-color: var(--denhaag-link-focus-color);
   outline: var(--denhaag-link-focus-outline);
 }
 
 .denhaag-link--hover,
 .denhaag-link:hover {
+  --denhaag-link-color: var(--denhaag-link-hover-color, var(--denhaag-link-color));
```

Comment:
Nice improvement. One small question: do we need to fallback the variable to itself? I think we can omit the fallback since it would already be set.

Reply:
Furthermore, while I get the the idea here and it would be nice if this worked, CSS variables do not support falling back to its own value. When `--denhaag-link-hover-color` is undefined this would result in `--denhaag-link-color` also becoming undefined, regardless of its previous value.

Reply:
@bddjong & @mjcarsjens , i have removed the fallback since i noticed it's indeed the same as the previous which is indeed not supported by the browser.